### PR TITLE
add SECURITY_STRICT macro for build options

### PR DESCRIPTION
**Pull request by jhou5**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -15,3 +20,6 @@ message(
     "\nThis script will build third party components licensed under various open source licenses into your container images.\n"
     "The terms under which those components may be used and distributed can be found with the license document that is provided with those components.\n"
     "Please familiarize yourself with those terms to ensure your distribution of those components complies with the terms of those licenses.\n\n")
+
+execute_process(COMMAND printf "\n-- Setting: SECURITY_STRICT=${SECURITY_STRICT}\n")
```
Comment (1480c1) [inline CMakeLists.txt]:
```suggestion
message(STATUS "Setting: SECURITY_STRICT=${SECURITY_STRICT}")
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -15,3 +20,6 @@ message(
     "\nThis script will build third party components licensed under various open source licenses into your container images.\n"
     "The terms under which those components may be used and distributed can be found with the license document that is provided with those components.\n"
     "Please familiarize yourself with those terms to ensure your distribution of those components complies with the terms of those licenses.\n\n")
```
Comment (1480c1) [inline CMakeLists.txt]:
```suggestion
    "Please familiarize yourself with those terms to ensure your distribution of those components complies with the terms of those licenses.\n\n"
    "\n-- Setting: SECURITY_STRICT=${SECURITY_STRICT}\n")
```

Just append it to here

File: CMakeLists.txt (hunk 1 above)
Comment (1480c1) [inline CMakeLists.txt]:
```suggestion
```

File: doc/cmake.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+
+Use the following definitions to customize the building process:   
+- **SECURITY_STRICT**: Specify the secuirty strict build options: ```Yes``` or ```No```. Default is ```No```. Currently secuirity strict build disable LAME from the images.
```
Comment (1480c1) [inline doc/cmake.md]:
```suggestion
- **SECURITY_STRICT**: Specify the security strict build options: `Yes` or `No`. Default is `No`. Currently security strict will disable building LAME from the images.   
```

Comment (jhou5) [reply doc/cmake.md]: Correct typo. Thanks!

File: CMakeLists.txt (hunk 2 above)
Comment (jhou5) [reply CMakeLists.txt]: Updated. Thanks

**Pull request merged**